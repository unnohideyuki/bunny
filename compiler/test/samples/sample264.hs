main = do print $ words "ab cde fgh i jk"
          putStrLn $ unwords ["abcd", "efgh"]
