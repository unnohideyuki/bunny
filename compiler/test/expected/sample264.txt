["ab","cde","fgh","i","jk"]
abcd efgh
